Source/WebCore/platform/mediastream/cocoa/AudioMediaStreamTrackRendererInternalUnit.cpp

```diff
@@ -210,19 +210,17 @@ void LocalAudioMediaStreamTrackRendererInternalUnit::createAudioUnitIfNeeded()
         return;
     }
 
-    if (!m_outputDescription) {
```

Comment:
The current contract is that m_outputDescription never changes throughout the lifetime of the internal unit.
Otherwise, the risk is that WebProcess and GPUProcess output descriptions will get out of sync.